variables.tf

```diff
@@ -53,19 +53,37 @@ variable "managed_identity_object_id" {
 }
 
 variable "appinsights_location" {
-  type        = "string"
+  type        = string
   default     = "West Europe"
   description = "Location for Application Insights"
 }
 
 variable "application_type" {
-  type = "string"
+  type = string
   default = "Web"
   description = "Type of Application Insights (Web/Other)"
 }
 
 variable "queue_max_delivery_count" {
-  type        = "string"
+  type        = string
   default     = "10"
   description = "Queue message max delivery counter. Extracted to variable so it can be assigned to application environment."
+}
+
+variable "team_name" {
+  type        = string
+  description = "Team name"
+  default     = "ECM Project"
+}
+
+variable "team_contact" {
+  type        = string
+  description = "Team contact"
+  default     = "#ethos-repl-service"
+}
+
+variable "destroy_me" {
```

Comment:
Lol, why this ?

Reply:
haha, a couple of projects have that variable!! not sure the reason.
https://github.com/hmcts/rd-shared-infrastructure/blob/master/variables.tf
https://github.com/hmcts/div-shared-infrastructure/blob/master/variables.tf